src/v/archival/ntp_archiver_service.cc

```diff
@@ -145,14 +145,21 @@ ss::future<> ntp_archiver::upload_loop() {
               _rtclog.error,
               "Failed to upload {} segments out of {}",
               result.num_failed,
-              result.num_succeded + result.num_failed);
+              result.num_succeded + result.num_failed + result.num_cancelled);
         } else if (result.num_succeded != 0) {
             vlog(
               _rtclog.debug,
               "Successfuly uploaded {} segments",
               result.num_succeded);
         }
 
+        if (result.num_cancelled != 0) {
+            vlog(
+              _rtclog.debug,
+              "Cancelled upload of {} segments",
+              result.num_succeded);
```

Comment:
@abhijat did you mean to print `result.num_cancelled` instead of `num_succeded`?

Reply:
Yes, good point, this was a bug. I had to fix it in in a later pr https://github.com/redpanda-data/redpanda/pull/5769